kubeflow/core/tensorboard.libsonnet

```diff
@@ -0,0 +1,327 @@
+{
+  // Parameters are intended to be late bound.
+  params:: {
+    name: null,
+    numGpus: 0,
+    labels: {
+      app: $.params.name,
+    },
+    modelName: $.params.name,
+    modelPath: null,
+    // TODO(jlewi): We should probably default to a string.
+    // We might also want to have a separate boolean to indicate whether or not to use the httpProxy.
+    httpProxyImage: 0,
+
+    serviceType: "ClusterIP",
+
+    // If users want to override the image then can override defaultCpuImage and/or defaultGpuImage
```

Comment:
defaultCpuImage and defaultGpuImage and are for TF serving. We should change them.

We should probably have a defaultTB image which could be one of the stock TensorFLow images containing TB.